README.md

```diff
@@ -88,7 +88,7 @@ steps:
 
 #### Supported version syntax
 The `java-version` input supports an exact version or a version range using [SemVer](https://semver.org/) notation:
-- major versions: `8`, `11`, `16`, `17`
+- major versions: `8`, `11`, `16`, `17`, `21`
 - more specific versions: `17.0`, `11.0`, `11.0.4`, `8.0.232`, `8.0.282+8`
```

Comment:
Does this also add support for specific versions? ie : `21.0.1+12` or will this be included in a subsequent change?

Reply:
That's only the documentation, but yes more generally the minor and patch versions will work